src/app/JSON/componentAwesomeSelective.json

```diff
@@ -1,6 +1,7 @@
 {
       "label": "AwesomeSelective",
       "type": 3,
+      "initialCase": "Lock",
```

Comment:
So when we set a branch to be default it should have a gaurd of the text default i think..